docs/es/week12/12-3.md

```diff
@@ -0,0 +1,546 @@
+
+<!-- For implementation, we can speed up computation by stacking all the $\vect{W}$'s into one tall $\vect{W}$ and then calculate $\vect{q}, \vect{k}, \vect{v}$ in one go: -->
+Para la implementación, podemos acelerar los cálculos agrupando todo los $\vect{W}$'s en un vector $\vect{W}$, y después calcular $\vect{q}, \vect{k}, \vect{v}$ de una sola vez:
+
+$$
+\begin{bmatrix}
+\vect{q} \\
+\vect{k} \\
+\vect{v}
+\end{bmatrix} =
+\begin{bmatrix}
+\vect{W_q} \\
+\vect{W_k} \\
+\vect{W_v}
+\end{bmatrix} \vect{x} \in \mathbb{R}^{3d}
+$$
+
+<!-- There is also the concept of "heads". Above we have seen an example with one head but we could have multiple heads. For example, say we have $h$ heads, then we have $h$ $\vect{q}$'s, $h$ $\vect{k}$'s and $h$ $\vect{v}$'s and we end up with a vector in $\mathbb{R}^{3hd}$: -->
+Existe también el concepto de "cabezas". Hemos visto previamente un ejemplo con una sola cabeza, pero es posible tener múltiples cabezas. Por ejemplo, digamos que tenemos $h$ cabezas. Entonces tenemos $h$ $\vect{q}$'s, $h$ $\vect{k}$'s y $h$ $\vect{v}$'s y terminamos con un vector en $\mathbb{R}^{3hd}$:
+
+$$
+\begin{bmatrix}
+\vect{q}^1 \\
+\vect{q}^2 \\
+\vdots \\
+\vect{q}^h \\
+\vect{k}^1 \\
+\vect{k}^2 \\
+\vdots \\
+\vect{k}^h \\
+\vect{v}^1 \\
+\vect{v}^2 \\
+\vdots \\
+\vect{v}^h
+\end{bmatrix} =
+\begin{bmatrix}
+\vect{W_q}^1 \\
+\vect{W_q}^2 \\
+\vdots \\
+\vect{W_q}^h \\
+\vect{W_k}^1 \\
+\vect{W_k}^2 \\
+\vdots \\
+\vect{W_k}^h \\
+\vect{W_v}^1 \\
+\vect{W_v}^2 \\
+\vdots \\
+\vect{W_v}^h
+\end{bmatrix} \vect{x} \in \R^{3hd}
+$$
+
+<!-- However, we can still transform the multi-headed values to have the original dimension $\R^d$ by using a $\vect{W_h} \in \mathbb{R}^{d \times hd}$. This is just one possible way to implement the key-value store. -->
+Sin embargo, aún es posible transformar los valores multi-cabeza para que tengan la dimensión original $\R^d$ usando un $\vect{W_h} \in \mathbb{R}^{d \times hd}$. Esta es sólo una de las posibles maneras de implementar el almacén de clave/valor.
+
+
+<!-- ## [The Transformer](https://www.youtube.com/watch?v=f01J0Dri-6k&t=2114s) -->
+## [El Transformador](https://www.youtube.com/watch?v=f01J0Dri-6k&t=2114s)
+
+<!-- Expanding on our knowledge of attention in particular, we now interpret the fundamental building blocks of the transformer. In particular, we will take a forward pass through a basic transformer, and see how attention is used in the standard encoder-decoder paradigm and compares to the sequential architectures of RNNs. -->
+Con el fin de expandir nuestro conocimiento sobre atención, interpretaremos ahora los bloques fundamentales del transformador. En particular, haremos un recorrido de principio a fin de un transformador básico, y veremos cómo se usa la atención en el paradigma estándar del codificador-decodificador, y cómo se compara esto con las arquitecturas secuenciales de las RNRs (Redes Neuronales Recurrentes).
```

Comment:
I have a question here. I've seen in the wiki for other languages that acronyms shouldn't be changed:
e.g. [https://github.com/Atcold/pytorch-Deep-Learning/wiki/Italian-translation#rules](https://github.com/Atcold/pytorch-Deep-Learning/wiki/Italian-translation#rules)
Are we following this rule in the Spanish translation as well?

Reply:
I wasn't aware of that rule. Thanks for bringing it up. I think it makes sense to use it for consistency with other translations.